### PR TITLE
Fix for PyroTaskProvider and package.json cleanup

### PR DESCRIPTION
**Pull request by rjstone**
task caching code broken, fixed now
- provideTasks() should not be async.
- Removed that and it works.
- Renamed cache property to something better.
project,json cleanup
- Remove orphaned contributes etc.
- Add myself and fireundibh as contributors

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,22 +1,47 @@
-version: 2.0.0.{build}
+version: build.{build}
```
Comment (joelday) [inline appveyor.yml]: Exclude these changes (for now).

File: src/papyrus-lang-vscode/src/features/PyroTaskProvider.ts [hunk 2]
```diff
@@ -34,20 +34,20 @@ export class PyroTaskProvider implements TaskProvider, Disposable {
...
-        fsw.onDidChange(() => this._ppjPromise = undefined);
-        fsw.onDidCreate(() => this._ppjPromise = undefined);
-        fsw.onDidDelete(() => this._ppjPromise = undefined);
+        fsw.onDidChange(() => this._taskCachePromise = undefined);
+        fsw.onDidCreate(() => this._taskCachePromise = undefined);
+        fsw.onDidDelete(() => this._taskCachePromise = undefined);
 
     }
 
-    async provideTasks(token?: CancellationToken): Promise<Task[]> {
```
Comment (joelday) [inline src/papyrus-lang-vscode/src/features/PyroTaskProvider.ts]: Oh jeeze, that was a sneaky bug.

File: appveyor.yml (hunk 1 above)
Comment (rjstone) [reply appveyor.yml]: That wasn't intended... I think I had some sort of merge problem.

Comment (rjstone) [reply appveyor.yml]: Ok, will change the symbol from ppj to projectFile

**Pull request merged**